Extension/src/LanguageServer/client.ts

```diff
@@ -809,7 +809,7 @@ class DefaultClient implements Client {
             util.setProgress(util.getProgressParseRootSuccess());
         } else if (message.endsWith("No Squiggles")) {
             util.setIntelliSenseProgress(util.getProgressIntelliSenseNoSquiggles());
-        } else if (message.endsWith("IntelliSense Fallback")) {
+        } else if (!this.configurationProvider && message.endsWith("IntelliSense Fallback")) {
```

Comment:
It's not quite this simple.  If the provider is absent or misbehaving, cpptools will be using c_cpp_properties.json to configure IntelliSense, so this message might still be needed.

The issue that @sherryyshi and her team actually want to solve is the first launch scenario (e.g. before the config provider is even added to c_cpp_properties.json), and this won't solve that.